kokoro/presubmit.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
-# On Kokoro instances /dev/kvm is owned by root:root. Let's fix the permissions
-# so that it's owned by group `kvm` as our scripts expect it to.
+# On Kokoro instances /dev/kvm, dev/vhost-vsock and dev/vsock are owned by root:root.
+# Let's fix the permissions so that it's owned by group `kvm` as our scripts expect it to.
```

Comment:
Is "Let's fix the permissions" something we still need to do? I don't think so. Maybe re-phrase to: "Set permissions so that /dev/kvm, /dev/vhost-sock, and /dev/vsock (?) are owned by group `kvm` ..."

Reply:
Updated the comments.